TestApp/app/screens/AnalyticsScreen.js

```diff
@@ -24,12 +27,18 @@ export default class AnalyticsScreen extends Component {
   }
 
   state = {
-    analyticsEnabled: false
+    analyticsEnabled: false,
+    isManualSessionEnabled: false
   }
 
   async componentDidMount() {
-    await this.refreshToggle();
-
+    NativeModules.TestAppNative.getSessionTrackerState().then((arr) => {
+      const isManualSessionEnabled = arr == 1;
+      this.setState({ isManualSessionEnabled });
```

Comment:
better to avoid local vars when possible. let's use: this.setState({ isManualSessionEnabled: isEnabled == 1 });
btw, why do we convert bool to int and int to bool again. Can we make `TestAppNative.getSessionTrackerState()` return `boolean`?